_posts/en/newsletters/2019-12-18-newsletter.md

```diff
@@ -0,0 +1,238 @@
+  newsletters ([#72][news72 bech32], [#74][news74 bech32], and
+  [#76][news76 bech32]) where it's possible to add or remove a `q`
+  character immediately before a `p` character at the end of a bech32
+  string.  Wuille's [analysis][wuille bech32 analysis] shows that this
+  is the only exception to bech32's expected error detection properties
+  and that "changing one constant in bech32 would resolve this issue."
+
+    Wuille plans to amend [BIP173][] to describe the weakness, propose a
+    change to limit existing bech32 address uses to either 20 byte or 32
+    byte witness program payloads, and define a modified version of
+    bech32 with the alternative constant for non-Bitcoin uses and for a
+    potential future where we want witness programs that are not 20
+    bytes or 32 bytes.
+
+- **Proposed changes to bip-ctv:** Jeremy Rubin [suggested][rubin ctv
+  update] several changes to the proposed soft fork addition of an
+  `OP_CHECKTEMPLATEVERIFY` ([CTV][topic covenants]) opcode.  Most
+  notably, the changes will do away with the restriction that template
+  used by CTV cannot be derived from other data via Bitcoin Script.
+  This change simplifies the change to the Script language as discussed
+  in [Newsletter #75][news75 ctv].  Neither this change nor any of the
+  other changes modifies CTV's behavior in a way that would
+  significantly affect previously-described usecases as far as we're
+  aware (but anyone who is aware of fundamental changes is encouraged to
+  discuss them on-list).
+
+- **Discussion of eclipse attacks on LN nodes:** Antoine Riard
+  [posted][riard eclipse] to the Lightning-Dev mailing list a
+  description of two attacks possible against LN users if they are
+  eclipse attacked and the attacker delays the relay of blocks.  An
+  eclipse attack occurs when all connections made by a full node or
+  lightweight client are controlled by a single attacker, as might
+  easily be the case when the attacker is an ISP or hacks into a user's
+  router.  This gives the attacker full control over what data the node
+  or client sends or receives.  In the first attack, the eclipse
+  attacker can confirm a revoked commitment transaction without the
+  honest user realizing it in time to submit the corresponding penalty
+  transaction, allowing the attacker to receive funds they previously
+  paid to the honest user.  In the second attack, the attacker prevents
+  the honest user from realizing that they need to broadcast the latest
+  commitment transaction because one of more of its HTLCs is about to
+  expire---this allows the attacker to steal the funds in the HTLC after
+  it does expire.
+
+    Both Riard's post and replies from [Matt Corallo][corallo eclipse]
+    and [ZmnSCPxj][zmn eclipse] discuss past and present work to make
+    full nodes and lightweight clients more resistant to eclipse
+    attacks.
+
+## Selected Q&A from Bitcoin StackExchange
```

Comment:
After having committed these 2 new sections, I can see it’s a bit weird formatting wise. The news and client sections have **bold** bullet titles and the SE and notable code sections have __link__ bullets.

<img width="1018" alt="Screen Shot 2019-12-17 at 12 00 58 PM" src="https://user-images.githubusercontent.com/1615772/71022043-351d9a00-20c5-11ea-94b7-312286a48fb9.png">

Im not offended enough to mandate a change, but perhaps we swap the SE/Client software sections to keep from alternative?

Reply:
Yeah, I think swapping them might look better, though I don't think it matters much either way.

Reply:
Pushed a commit to swap them.